run_dir/static/js/project_samples.js

```diff
@@ -653,10 +653,204 @@ function load_samples_table(colOrder) {
+                            var process_id = max_str(Object.keys(info_library['library_validation']));
+                            var validation_data = info_library['library_validation'][process_id];
+                            if (validation_data) {
+                              validation_data[column_id] = round_floats(validation_data[column_id], 2);
+
+                              // Remove the X from initial QC initials
+                              if(column_id == 'initials'){
+                                if(validation_data[column_id] !== '-'){
+                                  var sig = validation_data[column_id];
+                                  if(sig.length == 3 && sig[2] == 'X'){
+                                    sig = sig.substring(0,2);
+                                  }
+                                  tbl_row += '<span class="badge bg-secondary" data-toggle="tooltip" title="Original signature: '+validation_data[column_id]+'">'+sig+'</span><br>';
+                                }
+                                else{
+                                  tbl_row += '-<br>';
+                                }
+                              }
+
+                              // Everything else
+                              else {
+                                tbl_row += auto_format(validation_data[column_id], true);
+                              }
+                            }
+                          }
+                          tbl_row += '</td>';
+                        });
+                      }
+                      else if (value[2] == "pre-prep-library-validation-columns" && info['library_prep'] !== undefined) {
+                        tbl_row += '<td class="' + column_id + '">';
+                        var libs = Object.keys(info['library_prep']).sort();
+                        $.each(libs, function(idx, library){
+                          info_library=info['library_prep'][library];
+                          if ('pre_prep_library_validation' in info_library) {
+                            // We only want to show up the LIMS process ID with the higher number (the last one)
+                            var process_id = max_str(Object.keys(info_library['pre_prep_library_validation']));
+                            var validation_data = info_library['pre_prep_library_validation'][process_id];
+                            if (validation_data) {
+                              validation_data[column_id] = round_floats(validation_data[column_id], 2);
+                              tbl_row += auto_format(validation_data[column_id]);
+                            }
+                          }
+                        });
+                        tbl_row += '</td>';
+                      }
+      });
+      tbl_row += '</tr>';
+      tbl_body += tbl_row;
+    });
+    last_sample = sample;
+  }
```

Comment:
I found a 🐛 !

If you click on another button, they won't be activated because the "library validation"-button is still active 😬 

I guess I need something like `$('#library_validation').removeClass('active')` here?

But it breaks the coloring of the table 🙄

Reply:
Nevermind, it works now, the button was just stuck I guess 😆